_capabilities/cognitivesalience.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Cognitive Models for Common Ground Modeling"
+excerpt: "We model both the AI and the human performer in a common modeling framework and use cognitive salience to reveal their respective mental models"
+tags: # Select from this set
+  - Autonomy
+  - Computer Vision
+  - Reinforcement Learning
+  - Human-Machine Teaming
+  - Saliency
+   
+submission_details:
+  resources: # List any resources associated with the contribution. Not all sections are required
+    papers:
+      - https://iccm-conference.neocities.org/2020/papers/Contribution_255_final.pdf
+      - https://baicsworkshop.github.io/pdf/BAICS_36.pdf
+      - https://iccm-conference.neocities.org/2019/proceedings/papers/ICCM2019_paper_53.pdf
+    demos:
+      - https://colab.research.google.com/drive/1tf5gYVab3GSpPsTUxp34PVxNCIbA19FC?usp=sharing
+   
+  authors:
```

Comment:
You can create superscripts associating authors and organizations like such:

```
authors:
  - Sterling Somers<sup>1</sup>
  - Constantinos Mitsopolous<sup>1</sup>
  - Christien Lebiere<sup>1</sup>
  - Peter Perolli<sup>2</sup>
  - Joel Schooler<sup>2</sup>
  - Robert Thomson<sup>3</sup>
```
This should render properly when displayed on the website.

Reply:
done